test/internals/reply.test.js

```diff
@@ -477,6 +477,35 @@ test('stream with content type should not send application/octet-stream', t => {
   })
 })
 
+test('stream without content type should not send application/octet-stream', t => {
+  t.plan(4)
+
+  const fastify = require('../..')()
+  const fs = require('fs')
+  const path = require('path')
+
+  const streamPath = path.join(__dirname, '..', '..', 'package.json')
```

Comment:
naming: this is not a streamPath, it's a filePath. let's keep consistent with the existing tests